crates/ruff/src/registry.rs

```diff
@@ -258,31 +258,32 @@ ruff_macros::register_rules!(
     rules::flake8_annotations::rules::MissingReturnTypeClassMethod,
     rules::flake8_annotations::rules::AnyType,
     // flake8-2020
-    rules::flake8_2020::rules::SysVersionSlice3Referenced,
-    rules::flake8_2020::rules::SysVersion2Referenced,
+    rules::flake8_2020::rules::SysVersionSlice3Reference,
+    rules::flake8_2020::rules::SysVersion2Reference,
     rules::flake8_2020::rules::SysVersionCmpStr3,
-    rules::flake8_2020::rules::SysVersionInfo0Eq3Referenced,
-    rules::flake8_2020::rules::SixPY3Referenced,
+    rules::flake8_2020::rules::SysVersionInfo0Eq3Reference,
+    rules::flake8_2020::rules::SixPY3Reference,
     rules::flake8_2020::rules::SysVersionInfo1CmpInt,
     rules::flake8_2020::rules::SysVersionInfoMinorCmpInt,
-    rules::flake8_2020::rules::SysVersion0Referenced,
+    rules::flake8_2020::rules::SysVersion0Reference,
     rules::flake8_2020::rules::SysVersionCmpStr10,
-    rules::flake8_2020::rules::SysVersionSlice1Referenced,
+    rules::flake8_2020::rules::SysVersionSlice1Reference,
     // flake8-simplify
     rules::flake8_simplify::rules::ManualDictLookup,
     rules::flake8_simplify::rules::DuplicateIsinstanceCall,
     rules::flake8_simplify::rules::CollapsibleIf,
     rules::flake8_simplify::rules::NeedlessBool,
+    // TODO(charlie): Rename.
     rules::flake8_simplify::rules::UseContextlibSuppress,
     rules::flake8_simplify::rules::ReturnInTryExceptFinally,
-    rules::flake8_simplify::rules::UseTernaryOperator,
+    rules::flake8_simplify::rules::ManualTernaryOperator,
```

Comment:
An if else block isn't really a manual ternary operator.

Reply:
Any ideas for alternate framings?

Reply:
I guess my main issue with the current name is that Python doesn't even have a ternary operator. An if expression is very similar but it's strictly speaking not a ternary operator.

Reply:
Maybe `IfElseBlockInsteadOfExpr`.